README.md

```diff
@@ -41,12 +41,12 @@ Require this script on a page with Editor.js.
 ```javascript
 const editor = new EditorJS({
   onReady: () => {
-    new DragDrop(editor);
+    new DragDrop(editor, '1px solid red');
   },
 });
 ```
 
-Select the block, drag the toolbar settings button and drop it at the desired position.
+Select the block, drag the toolbar settings button, and drop it at the desired position. The optional second parameter adds styles to the indicator border; otherwise, the default border style value is used.
```

Comment:
@MarioRodriguezS 
- Please write instructions on the second parameter in a separate paragraph.
- We could mention the default styles applied.
- Also, I would remove the second parameter from the Usage code example.